action/protocol/staking/vote_bucket.go

```diff
@@ -87,18 +87,18 @@ func (vb *VoteBucket) fromProto(pb *stakingpb.Bucket) error {
 		return err
 	}
 
-	createTime, err := ptypes.Timestamp(pb.GetCreateTime())
-	if err != nil {
+	if err := pb.GetCreateTime().CheckValid(); err != nil {
 		return err
 	}
-	stakeTime, err := ptypes.Timestamp(pb.GetStakeStartTime())
-	if err != nil {
+	createTime := pb.GetCreateTime().AsTime()
+	if err := pb.GetStakeStartTime().CheckValid(); err != nil {
```

Comment:
make it consistent:
```
if err := ts.CheckValid(); err != nil {
    return err
}
someTime := ts.AsTime()
```

Reply:
This is how I do, L90-L93 are a group, and L94-L97 are a group

Reply:
I can make it separate, so it seems consistent.

Reply:
Like this 
https://github.com/iotexproject/iotex-core/blob/993d16c8f542e7dd297ae8f650c9903ec4c0c6e6/action/protocol/staking/vote_bucket.go#L90-L103